test/integration/smart_answer_flows/energy_grants_calculator_test.rb

```diff
@@ -48,13 +48,22 @@ class EnergyGrantsCalculatorTest < ActiveSupport::TestCase
           setup do
             add_response ' 4/07/1951'
           end
-          should "be eligible for winter fuel payment" do
-            assert_equal :winter_fuel_payment, current_state.calculator.age_variant
+
+          should "take you to help with bills outcome" do
+            assert_current_node :outcome_help_with_bills # outcome 1
           end
+        end
+
+        context "answer before 5th May 1953" do
```

Comment:
Did you mean to add this `context` in this commit? I was a bit surprised to see it grouped with the regeneration of responses and expected results file.